src/github-provider.ts

```diff
@@ -4,6 +4,10 @@
 // TODO: namespace provider zone; needs seneca-entity feature
 
 import { Octokit } from '@octokit/rest'
+import { Action, EntityCommand } from './types'
+import EntitiesData from './entitites.json'
+import { build_pattern } from './helper'
+import identify_handler from './identify-handler'
```

Comment:
prefer the `{ foo }` import style

Reply:
alright, done !
Is this preference a matter of maintainability ?